lib/galaxy/schema/drs/openapi.yaml

```diff
@@ -0,0 +1,1731 @@
+openapi: 3.0.3
```

Comment:
Does this file need to be included in the repo?

Reply:
Yeah - expanded diff for the same question 😆.